tests/bin/distrib-name/run.t

```diff
@@ -1,85 +1,80 @@
 
-    $ dune-release distrib --skip-lint
-    [-] Building source archive
-    dune-release: [ERROR] cannot determine distribution name automatically: add (name <name>) to dune-project
-    [1]
+  $ dune-release distrib --skip-lint > /dev/null
+  dune-release: [ERROR] cannot determine distribution name automatically: add (name <name>) to dune-project
+  [1]
 
-dune-release distrib --dry-run with no project name.
+dune-release distrib --dry-run with no project name should fail as well.
 
-    $ dune-release distrib --skip-lint --dry-run
-    [-] Building source archive
-    dune-release: [ERROR] cannot determine distribution name automatically: add (name <name>) to dune-project
-    [1]
+  $ dune-release distrib --skip-lint --dry-run > /dev/null
+  dune-release: [ERROR] cannot determine distribution name automatically: add (name <name>) to dune-project
+  [1]
 
 Add an uncommitted name to dune-project. (Because of a dune limitation
 this name must be one the .opam file names.)
 
-    $ echo "(name liba)" >> dune-project
+  $ echo "(name liba)" >> dune-project
 
 Run dune-release distrib with the uncomitted name in dune-project.
 
-    $ dune-release tag -y
-    [-] Extracting tag from first entry in CHANGES.md
-    [-] Using tag "0.42.0"
-    [+] Tagged HEAD with version 0.42.0
-    $ dune-release distrib --skip-lint
-    [-] Building source archive
-    dune-release: [WARNING] The repo is dirty. The distribution archive may be
-                            inconsistent. Uncommitted changes to files (including
-                            dune-project) will be ignored.
-    Error: The project name is not defined, please add a (name <name>) field to
-    your dune-project file.
-    dune-release: [ERROR] run ['dune' 'subst']: exited with 1
-    [3]
+  $ dune-release tag -y > /dev/null
+  $ dune-release distrib --skip-lint > /dev/null
+  dune-release: [WARNING] The repo is dirty. The distribution archive may be
+                          inconsistent. Uncommitted changes to files (including
+                          dune-project) will be ignored.
+  Error: The project name is not defined, please add a (name <name>) field to
+  your dune-project file.
+  dune-release: [ERROR] run ['dune' 'subst']: exited with 1
+  [3]
 
 Commit the change in dune-project and run distrib.
 
-    $ git add dune-project && git commit -m 'add name' > /dev/null
-    $ dune-release distrib --skip-lint
-    [-] Building source archive
-    [+] Wrote archive ...
-    
-    [-] Building package in ...
-    [ OK ] package(s) build
-    
-    [-] Running package tests in ...
-    [ OK ] package(s) pass the tests
-    
-    [+] Distribution for liba ...
-    [+] Commit ...
-    [+] Archive ...
+  $ git add dune-project && git commit -m 'add name' > /dev/null
+  $ dune-release distrib --skip-lint | make_dune_release_deterministic
+  [-] Building source archive
+  [+] Wrote archive _build/PKG(liba)-VER(0.42.0)-SINCE(1)-COMMIT(<deterministic>).tbz
```

Comment:
I'm a bit sceptical about this rewriting, it's a bit confusing. I wonder if it wouldn't be best to live the deterministic parts untouched and rewrites the bits that are with something a bit less invasive.

What's the original output we're rewriting here?

Reply:
You're right. Looking back I don't know why I did this wrapping. Maybe because I had fun with the `Re` combinators but of course that rewriting doesn't add anything.

The original line goes something like `_build/liba-0.42.0-1-gbadf00d`. I'll change the helper to just leave them be since everything but the commithash is reproducible anyway.

Reply:
Looks great now!